subworkflows/local/normalize.nf

```diff
@@ -0,0 +1,19 @@
+//
```

Comment:
This file should not exist anymore, it has already been updated and is now found in [subworkflows/local/normalize/main.nf](https://github.com/nf-core/nanostring/blob/dev/subworkflows/local/normalize/main.nf)